index.html

```diff
@@ -879,6 +879,33 @@ <h2 style="margin-top: 0;">EatersLab</h2>
 
 ## To wszystko - **dziękujemy**!
 
+---
+
+---
+.left-column[
+  ### Django
+]
+.right-column[
+
+* API
+  - GET, POST na większość modeli gotowa
+  - Mogą dojść nowe modele do dodania do API
+  - Autentykacja - WIP (Work In Progress)
+  
+* Generowanie dokumentacji API - gotowe
+  
+* Admin - gotowy
+
+* **Informacje o zajętości stołówki - WIP**
+
+* Część webowa
+  - statysyki - WIP
+  - strona główna - gotowa
+
+* Workflow + Heroku - gotowe
```

Comment:
write what is maybe to be done to be consistent with the resst of the presentation?